vagrant/packer/scripts/zfs-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+echo '==> Setting up ZFS...'
+
+# It's a VM, ignore fsync(2)
+echo '     * Changing logbias on zroot...'
+/sbin/zfs set logbias=throughput zroot
```

Comment:
Can you add:

```
/sbin/zfs set sync=disabled zroot
```

too?  `sync=disabled` makes `fsync(2)` nops, `logbias=throughput` skips the ZIL and sends writes to the zpool.

Reply:
:+1: